CHANGELOG.next.toml

```diff
@@ -105,3 +105,8 @@ author = "david-perez"
 references = ["smithy-rs#2676", "smithy-rs#2685"]
 meta = { "breaking" = true, "tada" = false, "bug" = false }
 
+[[smithy-rs]]
+message = "The `config_bag` and `type_erasure` modules have been moved to the `aws-smithy-types` crate from the `aws-smithy-runtime-api` crate."
+references = ["smithy-rs#2735"]
+meta = { "breaking" = true, "tada" = false, "bug" = false, "target" = "client" }
+author = "ysaito1001"
```

Comment:
I don't think this needs a changelog entry, as it will only confuse customers since none of these exist in the currently released code.
```suggestion
```

Reply:
I didn't plan to add this till the last minute but the fact that the `aws-smithy-runtime-api` crate is available on [crates.io](https://crates.io/crates/aws-smithy-runtime-api) made me think I should add it to the changelog, even if things in the crate are currently under development. Maybe I was overthinking?